### PR TITLE
Update draft.md

### PR DESCRIPTION
**Pull request by KJSCostello**

### CODE REVIEW

**Comment by KJSCostello** [comment]
noticed mistake

File: draft.md [hunk 1]
```diff
@@ -102,7 +102,8 @@ Emotiq will follow strict KYC/AML procedures during all phases of the TGE. Funds
 * Build a world-class in-house research team; 
 * Explore the different implementations of smart contract languages and their usability, to accelerate blockchain adoption across the mass-market and enterprise; and, 
 * Provide venture funding for early projects on the Emotiq blockchain.
-* Emotiq will aggressively pursue development, acquire the best talent, and explore new solutions. Its vision is ambitious, with equally ambitious goals.
+
+Emotiq will aggressively pursue development, acquire the best talent, and explore new solutions. Its vision is ambitious, with equally ambitious goals.
```
Comment (joelreymont) [inline draft.md:106]: This looks good to me!

Comment (KJSCostello) [reply draft.md:106]: Yes yes yes

Comment (KJSCostello) [reply draft.md:106]: Let's commit

**Pull request merged**